libraries/chain/evaluators/proposal_create_evaluator.cpp

```diff
@@ -27,40 +28,18 @@ void proposal_create_evaluator::do_apply(const proposal_create_evaluator::operat
               "Proposal life time is not in range of ${min} - ${max} seconds.",
               ("min", SCORUM_PROPOSAL_LIFETIME_MIN_SECONDS)("max", SCORUM_PROPOSAL_LIFETIME_MAX_SECONDS));
 
-    FC_ASSERT(_committee_service.is_exists(op.creator), "Account \"${account_name}\" is not in committee.",
+    committee_i& committee_service = committee_accessor(db()).get_committee(op.operation);
```

Comment:
Why 'accessor'? It is unsuitable name maybe.

Reply:
agree lets find better one.